sympy/series/tests/test_limits.py

```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
 def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
```

Comment:
Maintain two line spacing between the testcases.

Reply:
I have made the modifications